packages/core/redux/tokens/actions.js

```diff
@@ -323,3 +316,37 @@ export const estimateBtcFee = (params, callback) => async () => {
   }
 }
 
+export const getWalletBalances = ({ wallet }) => {
+  const providersMap = {
+    [BLOCKCHAIN_ETHEREUM]: ethereumProvider,
+    [BLOCKCHAIN_BITCOIN]: btcProvider,
+    [BLOCKCHAIN_BITCOIN_CASH]: bccProvider,
+    [BLOCKCHAIN_BITCOIN_GOLD]: btgProvider,
+    [BLOCKCHAIN_LITECOIN]: ltcProvider,
+    [BLOCKCHAIN_WAVES]: wavesProvider,
+    [BLOCKCHAIN_NEM]: nemProvider,
+  }
+  return providersMap[wallet.blockchain].getAccountBalances(wallet.address)
+}
+
+export const formatBalances = ({ balancesResult, blockchain }) => {
```

Comment:
This is not an action. Please move this method to 'utils.js' or another appropriate file